### PR TITLE
chore: bump kommander/opsportal to image tag 1.149.77

### PR DESCRIPTION
**Pull request by natmegs**

### CODE REVIEW

**Comment by jaxesn** [comment]
I'm.pretty sure the version mapping is still being used so it would need updating to beta6.

File: stable/kommander/values.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-  tag: 1.149.63
+  tag: 1.149.77
   pullPolicy: IfNotPresent
 replicas: 1
 logoutRedirectPath: /ops/landing
 clusterPollingInterval: 3000
 hostPollingInterval: 3000
 clusterPollingThrottleFactor: 1
-kubernetesVersionsSelection: '[{"kubernetesVersion":"1.15.5","konvoyVersion":"v1.3.0-beta5","kubeaddonsVersion":"kommander-beta5"}]'
+kubernetesVersionsSelection: '[{"kubernetesVersion":"1.15.5","konvoyVersion":"v1.3.0-beta6","kubeaddonsVersion":"kommander-beta6"}]'
```
Comment (natmegs) [inline stable/kommander/values.yaml]: Can someone confirm that these are the right versions for beta6?

Comment (s12chung) [reply stable/kommander/values.yaml]: you probably know more than me. it changes this:

https://github.com/mesosphere/kommander/blob/bd720f9cea98b1dca728ac70f5c3fa22e8a76553/kommander/backend/src/config.ts#L27-L52

Comment (alejandroEsc) [reply stable/kommander/values.yaml]: https://github.com/mesosphere/konvoy/pull/1000/files seems like we are opting for `kommander-beta6-4` which i am not sure of. @jaxesn can i get some clarity here?

Comment (jaxesn) [reply stable/kommander/values.yaml]: Why do we need the -4? Can we just stick with beta6 to make this easy

Comment (shaneutt) [reply stable/kommander/values.yaml]: Yes we can just use it without the -4

**Pull request merged**